.github/workflows/compilation_on_nuttx.yml

```diff
@@ -65,6 +65,7 @@ jobs:
           "CONFIG_INTERPRETERS_WAMR=y\\nCONFIG_INTERPRETERS_WAMR_AOT=y\\nCONFIG_INTERPRETERS_WAMR_FAST=y\\n",
           "CONFIG_INTERPRETERS_WAMR=y\\nCONFIG_INTERPRETERS_WAMR_AOT=y\\nCONFIG_INTERPRETERS_WAMR_CLASSIC=y\\n",
           "CONFIG_INTERPRETERS_WAMR=y\\nCONFIG_INTERPRETERS_WAMR_AOT=y\\nCONFIG_INTERPRETERS_WAMR_LIBC_BUILTIN=y\\n",
+          "CONFIG_INTERPRETERS_WAMR=y\\nCONFIG_INTERPRETERS_WAMR_AOT=y\\nCONFIG_INTERPRETERS_WAMR_LIBC_WASI=y\\nCONFIG_NET=y\\nCONFIG_NET_SOLINGER=y\\nCONFIG_NET_TCP=y\\nCONFIG_NET_TCP_WRITE_BUFFERS=y\\nCONFIG_NET_SOCKOPTS=y\\n",
```

Comment:
Could you clarify how the `CONFIG_NET*` parameters are used? I don't see any reference to them in the source code.

Reply:
libc-wasi now requires setsockopt etc and thus requires these nuttx options enabled.

Reply:
Could you point me to a code where those parameters are used? Thank you

Reply:
they are used in nuttx repo. https://github.com/apache/incubator-nuttx